DDR4 OC Guide.md

```diff
@@ -94,39 +91,46 @@ You should always test with a variety of stress tests to ensure your overclock i
+* [Super Pi Mod v1.5 XS](https://www.techpowerup.com/download/super-pi/) - another memory-sensitive benchmark, but I haven't used it as much as AIDA64. 1M - 8M digits should be enough for a quick benchmark. You only need to look at the last (total) time, where lower is better.
 * [HWBOT x265 Benchmark](https://hwbot.org/benchmark/hwbot_x265_benchmark_-_1080p/) - I've heard that this benchmark is also sensitive to memory, but I haven't really tested it myself.
 * [PYPrime 2.x](https://github.com/monabuntur/PYPrime-2.x) - This benchmark is quick, and scales very well with CPU core clock, cache/FCLK, memory frequency, and timings
 
 # General RAM Info
 ## Frequency and Timings Relation
 * RAM frequency is measured in megahertz (MHz) or million cycles per second. Higher frequency means more cycles per second, which means better performance.
 * Esoteric note: People often refer to DDR4-3200 as being 3200**MHz**, however in reality the real frequency of the RAM is only 1600MHz. As data is transferred on both the rising clock edge and falling clock edge in DDR (Double Data Rate) memory, the real frequency of the RAM is half of the number of transfers it makes per second. DDR4-3200 transfers 3200 million bits per second, and so is 3200**MT/s** (MegaTransfers per second) operating at a frequency of 1600**MHz**.
-* RAM timings are measured in clock cycles or ticks. Lower timings mean less cycles to perform an operation, which means better performance.
+* RAM timings are measured in clock cycles or ticks. Lower timings mean fewer cycles to perform an operation, which means better performance.
   * The exception to this is tREFI, which is the refresh interval. As its name suggests, tREFI is the time between refreshes. While the RAM is refreshing it can't do anything, so you'd want to refresh as infrequently as possible. To do that, you'd want the time between refreshes to be as long as possible. This means you'd want tREFI as high as possible.
 * While lower timings may be better, this also depends on the frequency the RAM is running at. For example, DDR4-3000 CL15 and DDR4-3200 CL16 have the same latency, despite DDR4-3000 running at a lower absolute CL. This is because the higher frequency offsets the increase in CL.
 * To calculate the actual time in nanoseconds (ns) of a given timing: `2000 * timing / ddr_freq`.
   * For example, CL15 at DDR4-3000 is `2000 * 15 / 3000 = 10ns`.
   * Similarly, CL16 at DDR4-3200 is `2000 * 16 / 3200 = 10ns`.
 
 ## Primary, Secondary and Tertiary Timings
-* [Intel](https://i.imgur.com/hcKDkCc.png)
-* [AMD](https://i.imgur.com/Ie4LVtI.png)
-* RAM timings are split into 3 categories: primary, secondary and tertiary. These are indicated by 'P', 'S', and 'T' respectively.
+* Intel
+
+  ![image](https://user-images.githubusercontent.com/69487009/156914502-104b0cbd-bbc4-4c94-a583-426e84f732eb.png)
+
+* AMD
+
+  ![image](https://user-images.githubusercontent.com/69487009/156914518-5510c6a2-6187-41ee-8acd-a8f76a5590c4.png)
+
```

Comment:
Why did you change the URLs?

Reply:
This is just to embed the images.

Reply:
What do you mean by embed?

Reply:
People won't have to click the link in order to see the images and it'll appear in the document as an image. You can see this in preview mode.